spec/unit/installer/xcode/pods_project_generator/file_references_installer_spec.rb

```diff
@@ -280,6 +280,21 @@ class PodsProjectGenerator
           end
 
           #-------------------------------------------------------------------------#
+
+          describe 'Installation With Development Pods' do
+            before do
+              @project = Project.new(config.sandbox.project_path)
+              @project.add_pod_group('BananaLib', fixture('banana-lib'), true)
+            end
+
+            it 'sets the path of the Pod group to the installation root' do
+              @installer.install!
+              group = @project.group_for_spec('BananaLib')
+              group.path.should == fixture('banana-lib').relative_path_from(config.sandbox.root).to_s
```

Comment:
should we be explicit on the value we expect instead of calling a method?

Reply:
hmm sure can do, easy enough to change later if necessary